include/eve/module/core/regular/impl/any.hpp

```diff
@@ -26,13 +26,29 @@ any_(EVE_REQUIRES(cpu_), O const& opts, T const& v) noexcept
   else if (C::is_complete) return eve::any(eve::top_bits { v });
   else if constexpr( has_emulated_abi_v<T> )
   {
-    bool res = false;
+    if constexpr (relative_conditional_expr<C>)
+    {
+      std::ptrdiff_t begin = cond.offset(as<T>{});
+      std::ptrdiff_t end   = begin + cond.count(as<T> {});
```

Comment:
first/last or f/l.
Stepanov/Sean parent use those - good enough for Stepanov - good enough for you.

begin/end are bnames often used in adl - you don't want that